build_tools/kokoro/gcp_ubuntu/bazel/linux/x86/swiftshader/build_kokoro.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Given current conventions, this should be: `build_tools/kokoro/gcp_ubuntu/bazel/linux/x86-swiftshader/integrations/build_kokoro.sh` See also my comments on #2546

Reply:
Okay. Moved to that path.